server/oauth2.go

```diff
@@ -245,6 +245,8 @@ type idTokenClaims struct {
 	Groups []string `json:"groups,omitempty"`
 
 	Name string `json:"name,omitempty"`
+
+	Login string `json:"dex.coreos.com/login"`
```

Comment:
Login is a bit vague. Would a claim name with the word "connector" in it be better suitable?